specifyweb/specify/api.py

```diff
@@ -252,6 +252,53 @@ def collection_dispatch(request, model) -> HttpResponse:
 
     return resp
 
+def collection_dispatch_bulk(request, model) -> HttpResponse:
+    """
+    Do the same as collection_dispatch, but for bulk POST operations.
+    Call this endpoint with a list of objects of the same type to create.
+    This reduces the amount of API calls needed to create multiple objects, like when creating multiple carry forwards.
+    """
+    checker = table_permissions_checker(request.specify_collection, request.specify_user_agent, "read")
+
+    if request.method != 'POST':
+        return HttpResponseNotAllowed(['POST'])
+        
+    data = json.loads(request.body)
+    resp_objs = []
+    for obj_data in data:
+        obj = post_resource(
+            request.specify_collection,
+            request.specify_user_agent,
+            model,
+            obj_data,
+            request.GET.get("recordsetid", None),
+        )
+        resp_objs.append(_obj_to_data(obj, checker))
+
+    return HttpResponseCreated(toJson(resp_objs), content_type='application/json')
+
+def collection_dispatch_bulk_copy(request, model, copies) -> HttpResponse:
+    checker = table_permissions_checker(request.specify_collection, request.specify_user_agent, "read")
+
+    if request.method != 'POST':
+        return HttpResponseNotAllowed(['POST'])
+
+    data = json.loads(request.body)
+    data = dict(filter(lambda item: item[0] != 'id', data.items())) # Remove ID field before making copies
+    # resp_objs = [] # For now, we don't need to return the created objects to avoid sending large amounts of data 
```

Comment:
> For now, we don't need to return the created objects to avoid sending large amounts of data 

If we are concerned about response size, then the current endpoint would need to be modified and optimized for storage as well. 
It returns the exact same thing: an array of serialized resources. 

Either return an array of serialized resources here if we don't care about response size, or find a way to reduce the size on the current endpoint (possibly by just returning an array of ids?)